app/renderer/components/main/main.js

```diff
@@ -736,31 +744,23 @@ class Main extends React.Component {
           : null
         }
       </div>
-      <div className='mainContainer'>
-        <TransitionGroup className={cx({
```

Comment:
Since this is gone now, can we remove the package.json reference to `react-transition-group`?

Reply:
I was just about to do this, but then realised that this may make the merge a bit more of a pain, especially package-lock.json, across branches. Perhaps a separate PR unless you think that it won't be a pain?

Reply:
Separate PR is great 😄 👍 Let's leave it alone